packages/js/product-editor/src/components/product-mvp-feedback-modal/product-mvp-feedback-modal.tsx

```diff
@@ -18,28 +17,29 @@ import { Text } from '@woocommerce/experimental';
 import { __ } from '@wordpress/i18n';
 import { useDispatch } from '@wordpress/data';
 
+export type ProductMVPFeedbackModalProps = {
+	recordScoreCallback: (
+		checked: string[],
+		comments: string,
+		email: string
+	) => void;
+	onCloseModal?: () => void;
+	onSkipFeedback?: () => void;
+};
+
 /**
  * Provides a modal requesting customer feedback.
  *
- *
  * @param {Object}   props                     Component props.
  * @param {Function} props.recordScoreCallback Function to call when the results are sent.
  * @param {Function} props.onCloseModal        Function to call when user closes the modal by clicking the X.
  * @param {Function} props.onSkipFeedback      Function to call when user skips sending feedback.
  */
-function ProductMVPFeedbackModal( {
+export function ProductMVPFeedbackModal( {
 	recordScoreCallback,
 	onCloseModal,
 	onSkipFeedback,
-}: {
-	recordScoreCallback: (
-		checked: string[],
-		comments: string,
-		email: string
-	) => void;
-	onCloseModal?: () => void;
-	onSkipFeedback?: () => void;
-} ): JSX.Element | null {
+}: ProductMVPFeedbackModalProps ): JSX.Element | null {
```

Comment:
I don't think there's a case where `null` is returned in this component:

```suggestion
}: ProductMVPFeedbackModalProps ): JSX.Element {
```

Reply:
done